source/user-guide/export/index.rst

```diff
@@ -17,7 +17,9 @@ Export as a CSV file
 
 When a selected set of metadata records is exported as a CSV/TXT file, the following process takes place for each metadata record:
 
-- a brief summary of some of the elements from each selected metadata record is generated by applying the brief template from the metadata schema eg. for an iso19139 metadata record the brief template from ``GEONETWORK_DATA_DIR/config/schema_plugins/iso19139/present/metadata-iso19139.xsl`` would be applied to the metadata record.
+- a brief summary of some of the elements from each selected metadata record is generated by applying the brief template from the metadata schema. The main folder of the different available schema is  ``GEONETWORK_DATA_DIR/config/schema_plugins/``. 
+  In particular,  for an iso19139 metadata record the brief template from ``GEONETWORK_DATA_DIR/config/schema_plugins/iso19139/present/metadata.xsl`` would be applied to the metadata record.
```

Comment:
You can remove that line. It is not true anymore in 3.x.
For all schemas it goes directly to {schema]/layout/tpv-csv.xsl

Reply:
ok. I was supposing that it was in that way because the folder tree was completely different.